src/Form/FormAnalyzer.js

```diff
@@ -3,12 +3,12 @@ import { constants } from '../constants.js'
 import { matchingConfiguration } from './matching-configuration.js'
 import { getText, isLikelyASubmitButton } from '../autofill-utils.js'
 
-const negativeRegex = new RegExp(/sign(ing)?.?in(?!g)|log.?in|unsubscri|(forgot(ten)?|reset) (your )?password|password (forgotten|lost)/i)
-const positiveRegex = new RegExp(
+const loginRegex = new RegExp(/sign(ing)?.?in(?!g)|log.?in|unsubscri|(forgot(ten)?|reset) (your )?password|password (forgotten|lost)/i)
+const signupRegex = new RegExp(
```

Comment:
I've renamed this positive/negative to signup/login to make it more obvious what they do. They were pretty confusing before. The rationale of the naming dates back to when we had a binary Dax/No-Dax choice, but now things are more complex.